systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -97,11 +97,11 @@ public class AbstractUpgradeST extends AbstractST {
     protected File kafkaYaml;
 
     protected void makeSnapshots() {
-        coPods = DeploymentUtils.depSnapshot(clusterOperator.getDeploymentNamespace(), ResourceManager.getCoDeploymentName());
-        zkPods = PodUtils.podSnapshot(clusterOperator.getDeploymentNamespace(), zkSelector);
-        kafkaPods = PodUtils.podSnapshot(clusterOperator.getDeploymentNamespace(), kafkaSelector);
-        eoPods = DeploymentUtils.depSnapshot(clusterOperator.getDeploymentNamespace(), KafkaResources.entityOperatorDeploymentName(clusterName));
-        connectPods = PodUtils.podSnapshot(clusterOperator.getDeploymentNamespace(), connectLabelSelector);
+        coPods = DeploymentUtils.depSnapshot(io.strimzi.systemtest.Constants.CO_NAMESPACE, ResourceManager.getCoDeploymentName());
```

Comment:
```suggestion
        coPods = DeploymentUtils.depSnapshot(Constants.CO_NAMESPACE, ResourceManager.getCoDeploymentName());
```
(applies for all other places)

Reply:
This is a little bit complicated because we use import from `import io.strimzi.api.kafka.model.Constants;`. Thus, our `io.strimzi.systemtest.Constants` uses the same name of the class as from the API module. So I have two ideas:
1. Use our import of constants (i.e., `io.strimzi.systemtest.Constants`) and for others add prefix `io.strimzi.api.kafka.model`.
2. Or we can differentiate our Constants class and rename it to `ConstantsSTs` or something like that.
WDYT @im-konge ?

Reply:
`ConstantsSTs` seems to me like another ST class with tests (because of the suffix)
`TestConstants` would make more sense to me

Anyway, this is something for a completely different PR. I didn't see that other import, so let's keep it now as it is.